README.md

```diff
@@ -1459,6 +1460,24 @@ def finish_guided_session
   [foreman](https://github.com/ddollar/foreman) to manage them.
 <sup>[[link](#foreman)]</sup>
 
+## Auto loader rules
+
+Prefer nesting a class within a module instead of using module as part of class name:
+
+  ```ruby
+  module User
+    class Admin
+    end
+  end
+
+  class User::Admin
+  end 
+  ```
+
+If a class is nested within a module you do not have to provide the fully qualified class name.
```

Comment:
Could we add some of the information we researched? Isn't there more to this? As far as I remember, the real problem wasn't User::Admin vs module User/class Admin because it had something to do with classes with the same name but nested in different modules - wasn't that the reason why rails autoloader failed to load those classes?